spec/live/article_spec.rb

```diff
@@ -11,6 +11,12 @@
     expect(article.translations.map(&:locale)).to include("fr")
   end
 
+  it "can have labels", :vcr do
+    article.labels.create(name: "test-label")
+
+    expect(article.labels.map(&:name)).to include("test-label")
```

Comment:
Can we cleanup and delete the label after the test?

Reply:
> we could make the label name time related, eg "test-label-#{Time.to_i}" to avoid any possible conflict with existing labels.

We could, and this should work for live specs here, but since these tests use VCR (and we don't have Timecop as a dependency... not sure it'd be worth adding here yet?) each subsequent run that uses the cassettes would fail. Switching to using `create!` instead should error early if there is another label with the same name.

 > right now, the latest master is a beta version, so I'm not sure what's the best strategy to release this.
 
It looks like it's been a release candidate since September, so maybe we could cut the next version and add this to it? Or this could be backported to a prior minor version as a patch.

Reply:
VCR: I haven't looked much into how VCR is setup, but if the cassette is stored in GitHub... how is it reused? I don't think timecop is necessary at this point.  Ultimately, any solution that cleans after itself is fine by me.

Release: I would prefer to release the new version with the exact code of the beta version, after that, a new release won't be that hard.

Reply:
Also, this line: https://github.com/zendesk/zendesk_api_client_rb/actions/runs/3833571414/jobs/6525157576#step:4:70